<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Тестер Прогресс бара</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Программа</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="darkTheme">
         <property name="text">
          <string>Темная тема</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Нажимай</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="lightTheme">
         <property name="text">
          <string>Светлая тема</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>335</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="add10"/>
    <addaction name="del10"/>
    <addaction name="clearBut"/>
    <addaction name="closeBut"/>
    <addaction name="separator"/>
    <addaction name="aboutProg"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="add10">
   <property name="text">
    <string>Добавить 10%</string>
   </property>
  </action>
  <action name="del10">
   <property name="text">
    <string>Убрать 10%</string>
   </property>
  </action>
  <action name="clearBut">
   <property name="text">
    <string>Сброс</string>
   </property>
  </action>
  <action name="closeBut">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="aboutProg">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
